### DISCUSSION TITLE
Customizing Full-Screen Live Feed

### DISCUSSION THREAD

**Discussion by akivacp** [gtxaspec/wz_mini_hacks#658] [General]
Hey everyone,

I wanted to share a modification I made to the Wyze Mini Hacks camera's web interface, specifically regarding the car control feature. Personally, I found the car control page not very useful for my needs, but I did like having a live feed of my camera in the top corner of the car control interface. So, I tweaked the code a bit to display a full-screen live feed from the camera. Here's how you can do it:

```
<!DOCTYPE html>
<html>
<head>
  <style>
    body, html {
      margin: 0;
      padding: 0;
      height: 100%;
      overflow: hidden;
      background-color: black; /* Set the background color of the body to black */
    }
    #car_feed {
      width: 100%;
      max-height: 100vh;
      object-fit: contain;
    }
  </style>
</head>
<body>
  <img id="car_feed" src="/cgi-bin/jpeg.cgi?channel=1" />

  <script>
    setInterval(function() {
      var myImageElement = document.getElementById('car_feed');
      myImageElement.src = '/cgi-bin/jpeg.cgi?channel=1&_=' + new Date().getTime(); // Adding a timestamp to prevent caching
    }, 500); // Refresh every 500 milliseconds
  </script>
</body>
</html>

```

Feel free to give it a try and let me know if you have any suggestions or improvements!

**Comment by virmaior** [comment]
I don't think you need jquery for that bit of javascript.

**Reply by akivacp** [reply]
Updated
